src/getHumanAge.test.js

```diff
@@ -7,4 +7,36 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return [0, 0] when animalgAge are 0', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test('should return [0, 0] when animalAge < than first', () => {
```

Comment:
Did not quite understand the description of the test. Bigger than first what?

Reply:
Fixed.